lib/mocks/jobs.coffee

```diff
@@ -12,15 +12,25 @@ class MockJob extends EventEmitter
   intervalEmit: (event, timeout, data) ->
     interval = Math.floor timeout / (data.length + 1)
     @delayEmit event, (i + 1) * interval, el for el, i in data
+  # Takes a list of events and returns a new list of events sorted in increasing order, with each
+  # timeout being the offset from the previous event, such that the total offset for the event is
+  # the same.
+  @orderEvents: (events) ->
+    return events unless events.length > 1
```

Comment:
The logic doesn't work with an array of size 0, so I needed at least `events.length > 0`, and then I just did 1 anyway because the logic does nothing with an array of length 1. I could change it to when it's an empty array if you want